### ISSUE TITLE
`function-space-after` errors on new line after function

### ISSUE THREAD

**Issue by rtablada** [stylelint/stylelint#451]
When working with the `function-space-after` rule, a new line character is invalid after functions and only ' ' characters would work.

This can cause errors when a style guide also says to use compound rules and a max line length.
For instance take this background rule:

```
background: url(http://lorempixel.com/g/1000/1000/technics)
    no-repeat center center fixed;
```

This retains line-length and indentations while using a compound rule but gives an error for `function-space-after`.
Instead the current rule would give an error but the valid code would be >91 chars long:

```css
background: url(http://lorempixel.com/g/1000/1000/technics) no-repeat center center fixed;
```

I'm not sure if this would require a new `function-whitespace-after` rule to allow for similar functionality but allow more flexibility or maybe adding a new option for `any` or something similar that allows any form of whitespace instead of just a single space.

**Comment by davidtheclark** [comment]
I agree with the prescription: the same rule should check for *any* kind of whitespace after a function (I think there is no purpose in splitting up into `function-newline-after` and `function-space-after`, etc.). As for implementation: I think it could work if we go in two steps:

- Within v2, make `function-space-after` tolerate newlines. I would not consider this is not a breaking change.
- For v3, change the name to `function-whitespace-after`.

Seem reasonable?

**Comment by rtablada** [comment]
Seems good to me.




And for v3 I think maybe specifying valid chars for styleguides that may not allow split lines.



—

**Comment by davidtheclark** [comment]
@jeddy3 : What do you think about the renaming proposal for v3?

**Comment by jeddy3** [comment]
> What do you think about the renaming proposal for v3?

Sounds good to me.
